contracts/container/contract.go

```diff
@@ -13,9 +13,32 @@ import (
 	"github.com/nspcc-dev/neofs-contract/common"
 	cst "github.com/nspcc-dev/neofs-contract/contracts/container/containerconst"
 	"github.com/nspcc-dev/neofs-contract/contracts/nns/recordtype"
+	iproto "github.com/nspcc-dev/neofs-contract/internal/proto"
 )
 
 type (
+	// APIVersion represents NeoFS API protocol version.
+	APIVersion struct {
+		Major uint32
+		Minor uint32
+	}
+
+	// Attribute represents container attribute.
+	Attribute struct {
+		Key   string
+		Value string
+	}
+
+	// Info represents information about container.
+	Info struct {
+		Version       APIVersion
+		Owner         []byte // 25 bytes address
```

Comment:
We can use Hash160 already, 25-byte thing can be produced from it if needed, but normal NEP-11 owner is 20-byte.